Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -150,22 +150,18 @@ bool InputConfig::LoadConfig(InputClass type)
   }
   else
   {
-    // Only load the default profile for the first controller,
-    // otherwise they would all share the same mappings and default device
+    // Only load the default profile for the first controller and clear the others,
+    // otherwise they would all share the same mappings on the same and default device
```

Comment:
The addition of "on the same" doesn't make sense to me.

Reply:
On the same device, which is also the default device.

Reply:
I see what you mean then... But I think it just makes the sentence more confusing.

Reply:
It could read "with the same default device".

Reply:
Addressed